Assignment 1/src/main/java/com/cezaram28/Assignment1/repository/jdbc/JdbcTagRepository.java

```diff
@@ -23,10 +23,12 @@
 
     @Override
     public Tag save(Tag tag) {
-        if (tag.getId() == null) {
+        List<Tag> t = template.query("SELECT * FROM tag WHERE name=?", ((resultSet, i) -> new Tag(resultSet.getInt("id"), resultSet.getString("name"))), tag.getName());
```

Comment:
Naming... "t" is a very bad variable name. "tags" would be more appropriate or "matchingTags".